src/serde/parse_atom.rs

```diff
@@ -9,40 +9,44 @@ const MAX_SINGLE_BYTE: u8 = 0x7f;
 /// decode the length prefix for an atom. Atoms whose value fit in 7 bits
 /// don't have a length prefix, so those should be handled specially and
 /// never passed to this function.
```

Comment:
you should probably update this comment now that it also returns the size of the length-prefix